### PR TITLE
Update main.swift

### PR DESCRIPTION
**Pull request by jekkaii**
Changes with Declaration of arrays and naming conventions. Please Review other classes.

### CODE REVIEW

**Comment by jekkaii** [comment]
Please Review the spacings and indentations.

File: finals_code/main.swift [hunk 1]
```diff
@@ -22,13 +22,13 @@
 // -~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~
 
 // create a class named Account
-class account 
+class Account 
 {
  var username:String         // declare username variable [String]
```
Comment (JoefreyToriano) [inline finals_code/main.swift:27]: The comments are redundant. Refrain from commenting the same phrase on the same methods.

File: finals_code/main.swift [hunk 2]
```diff
@@ -22,13 +22,13 @@
...
  var password:String         // declare password variable [String]
-var firstName: String       // declare password firstname [String]
- var lastName: String        // declare password lastname [String]
```
Comment (JoefreyToriano) [inline finals_code/main.swift:30]: delete unnecessary comments

**Review by JoefreyToriano — COMMENTED** [review]
The changes are good, but check on your comments as well.

**Comment by JoefreyToriano** [comment]
consider my review.

**Pull request merged**